cscs-checks/libraries/gridtools/gridtools_tests.py

```diff
@@ -0,0 +1,203 @@
+import os
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.parameterized_test(['vertical_advection_dycore_naive'],
+                        ['vertical_advection_dycore_mc'],
+                        ['simple_hori_diff_naive'], ['simple_hori_diff_mc'],
+                        ['vertical_advection_dycore_cuda'],
+                        ['simple_hori_diff_cuda'])
+class GridToolsCheck(rfm.RegressionTest):
+    def __init__(self, variant):
+        super().__init__()
+
+        # Check if this is a device check
+        is_cuda_test = "cuda" in variant
+
+        self.descr = 'GridTools test base'
+
+        self.valid_prog_environs = ['PrgEnv-gnu']
+        self.modules = ['CMake', 'Boost']
+        if self.current_system.name is 'daint':
+            self.modules.append('gcc/5.3.0')
+        else:
+            self.modules.append('gcc')
+        if is_cuda_test:
+            if self.current_system.name is 'daint':
```

Comment:
You don't need to make the system distinction if you use `craype-accel-nvidia60` instead of the `cudatoolkit` module.

Reply:
Done.